composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "asgorobets/soap-extension",
+  "name": "behat/soap-extension",
```

Comment:
Also, changing the vendor name will affect installation instructions in README.md